File: ruby/bubble_sort/bubble_sort.rb
def sort(list)
  counter = list.size - 1
  while counter !=0 do
    for i in 1..counter  do
      if list[i - 1] > list[i]
        list[i], list[i - 1] = list[i - 1], list[i]
      end
    end
    counter -= 1
  end
  list
end

File: python/bubble_sort/bubble_sort.py
def sort(list):
  counter = len(list) - 1
  while counter !=0:
    for i in range(1, counter):
      if list[i - 1] > list[i]:
        list[i], list[i - 1] = list[i - 1], list[i]
    counter -= 1
  return list
